src/main/java/reposense/util/Constants.java

```diff
@@ -16,7 +16,8 @@
     public static final String LOG_SPLITTER = "\\|";
     public static final DateFormat GIT_ISO_FORMAT = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
     public static final DateFormat REPORT_NAME_FORMAT = new SimpleDateFormat("yyyy-MM-dd-HH-mm-ss");
-    public static final DateFormat GIT_LOG_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+    public static final DateFormat GIT_LOG_SINCE_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'00:00:00+08:00");
```

Comment:
Not sure if we should standardize the timezone to GMT +8 for all date operations.
It is reasonable to use it for system testing but more research has to been done for users of other timezone.

I'll try to test it to see if it cause any problems.